src/v2/Apps/ViewingRoom/__tests__/ViewingRoomClosedApp.jest.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react"
```

Comment:
This test file should be co-located with its related source file, in a `__tests__` folder -- also, no need for `App` in the file name.

Reply:
Well, the idea here is that since the change are made in App file itself - I just wanted a parallel test for the ClosedRoomApp. Ideally we would have 2 parallel contexts in the `ViewingRoomApp` test but I was not sure how to do it with the mocking the Query properly. So figured I would just add a parallel file. But I guess I can try to do 2 contexts in the original test file. I can see how it's confusing here.

Reply:
Yah thinking further this file can be collapsed into main test file or condensed to only test the child, prob doesn't need to stand alone.